AndroidHomeApplication/app/src/main/java/com/example/androidhomeapplication/fragments/movieDetails/FragmentMovieDetails.kt

```diff
@@ -48,6 +45,39 @@ class FragmentMovieDetails : Fragment(R.layout.fragment_movie_details) {
         }
     }
 
+    private fun setupObservers() {
+
+        viewModel.movie.observe(
+            viewLifecycleOwner,
+            {result: DataResult<Movie> ->
+                when (result) {
+                    is DataResult.Default -> {
```

Comment:
u can enumerate like:
when (result) {
                     is DataResult.Default ,
                     is DataResult.Loading -> {}
                     
                     if u need same action for multiple variants